src/xcode/ENA/ENA/Source/Scenes/CreateAntigenTestProfile/CreateAntigenTestProfileViewController.swift

```diff
@@ -49,15 +118,290 @@ class CreateAntigenTestProfileViewController: UIViewController, FooterViewHandli
 	func wasAttemptedToBeDismissed() {
 		dismiss()
 	}
+	
+	// MARK: - Protocol UITextFieldDelegate
 
-	// MARK: - Public
-
-	// MARK: - Internal
+	func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+		if textField.returnKeyType == .next {
+			if let cell = tableView.cellForRow(at: IndexPath(row: 1, section: 0)) as? InputCell, let currentIndex = cell.textFields.firstIndex(of: textField), (currentIndex + 1) < cell.textFields.count {
+				cell.textFields[currentIndex + 1].becomeFirstResponder()
+			} else {
+				textField.resignFirstResponder()
+			}
+		} else {
+			textField.resignFirstResponder()
+		}
+		return true
+	}
 
 	// MARK: - Private
 
 	private let viewModel: CreateAntigenTestProfileViewModel
 	private let didTapSave: () -> Void
 	private let dismiss: () -> Void
+	
+	private var dateFormatter: DateFormatter!
```

Comment:
I think we don't need it. You can use `DateFormatter.localizedString(from: date, dateStyle: .medium, timeStyle: .medium)`  for the default output